services/pkgreview/package-rating.service.js

```diff
@@ -0,0 +1,80 @@
+'use strict'
```

Comment:
We typically (maybe even always) are able to utilize a single class to provide both the rating and star flavors of a badge. This eliminates a lot of the boilerplate code and makes it a bit easier to maintain. We should do the same here.

See a couple other service classes that provide the same types of badges for references:

https://github.com/badges/shields/blob/master/services/visual-studio-marketplace/visual-studio-marketplace-rating.service.js

https://github.com/badges/shields/blob/master/services/spiget/spiget-rating.service.js

Reply:
Okay! I will take a look and will try to merge them both!